musileko/client/gl/texture.zig

```diff
@@ -214,33 +248,72 @@ pub const PixelChannels = enum(c_uint) {
             .r => 1,
             .rg => 2,
             .rgb => 3,
+            .srgb => 3,
             .rgba => 4,
+            .srgb_alpha => 4,
             .depth => 1,
         };
     }
 
+    pub fn glType(comptime self: Self) c_uint {
```

Comment:
This was changed to a separate fn specifically so that srgb and rgb can map to the same glType, same for the PixelComponent fn